### DISCUSSION TITLE
VSS,MRT , pharmacokinetic steady state

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#1088] [Questions & Problems]
Can pksim calculate vss and mrt in a multiple dose administration (or at pharmacokinetic steady state condition)?

**Comment by AndreDlm** [comment]
Hi @ranamjz, in the manual you'll find an overview of all PK parameters that are calculated in PK-Sim simulations:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-analysis-view

**Reply by ranamjz** [reply]
Thank you very much. Somewhere in the manual mentions that Vss and Vd can be calculated via physiological parameters , where is that option in the software because by the looks of it , it is calculating them  using formulas

**Reply by msevestre** [reply]
Calculation of Vss and Vd is performed as described in the link that @AndreDlm provided.
If there is a mention of those parameters calculated via physiological parameters, this is probably wrong. Can you provide me with the page where this is mentioned please?
